Access_Control_Policy.md

```diff
@@ -0,0 +1,68 @@
+# ACCESS CONTROL POLICY (DRAFT)
+
+## Classification Level
+Publicly available
+
+## Review Information
+
+__Mandatory Review Period__
+
+Yearly
+
+__Date of Last Review__
+
+N/A
+
+## Introduction
+
+An access control policy helps employees and external parties understand who is allowed to access what client and user data 
+and under what circumstances and conditions they can access that data.
+
+### Goal Statement
+
+At present, we are a small company (literally everyone in the company knows everyone else on a first name basis). The 
+unfortunate reality is that, no matter how much we may know, like, or trust an individual we know personally, they have the 
+capacity to take harmful actions. Beyond that, there is risk that an employee's identity (from a system perspective) may be 
+compromised allowing an attacker the same level of access as an employee. In general, we mitigate these kinds of risks by 
+collecting as little data as possible so that, even in the event of a breach, nothing of value would be lost. However, we do
+offer a service whose value is in its availability and providing too much access to everyone could result in a loss of that 
+availability to our clients. To that end we need some basic guidelines indicating who should have access to what, and how,
+when, and why that access should change.
+
+### Background Statement 
+
+We, at Fonticons, know our culture but it is entirely reasonable for others planning to use our technology to desire to 
+understand our goals and commitments around security and privacy. Some of our policies have been de facto in nature, e.g.,
+you simply can't get access to something because so-and-so wouldn't give it to you. While this is arguably effective for a 
+company of our size, having a de jure policy that others can consider is appropriate for the space in which we work. 
+
+## Definitions 
+
+### Terms
+
+* The word "we" shall mean Fonticons Inc, all Fonticons employees and any individuals contracting with Fonticons to complete work.
+* Employee shall mean an individual directly employed by Fonticons Inc and all contractors, consultants, temporary employees, or business partners.
+* Client shall mean a person or entity who installs or configures part or all of the Font Awesome free product/service for use on a website or product not owned or otherwise controlled by Fonticons.
+* Pro client shall mean a person or entity who installs or configures part or all of the Font Awesome paid product/service for use on a website or product not owned or otherwise controlled by Fonticons.
```

Comment:
If we do, we should probably prioritize threat modeling those other solutions as well.  It isn't a problem, I'd just been tentatively focused on Font Awesome.  We could publish these and then update them after I'd done the threat models.  It would also indicate if there were other considerations I was previously unaware of.

Reply:
I've thought about it more in the last 7 minutes and have decided that it should mean all products/services.